AWS Lambda failed

# 🐛 Description

handler fails when reading files?

# 🔢 To reproduce


# ✌🏽 Expected behaviour


# 📷 Screenshots


# 💻 Traceback

```
[ERROR] Level1BException: Failed to process ops-payload-level1a-v0.3/2022/12/29/MATS_OPS_Level0_VC1_APID100_20221229-142806_20221230-130153.parquet
Traceback (most recent call last):
  File "/var/task/level1b.py", line 110, in lambda_handler
    raise Level1BException(msg) from err

```

--- a/l1b_lambda/.dockerignore
+++ b/l1b_lambda/.dockerignore
@@ -1,0 +1,6 @@
+.mypy_cache
+**/__pycache__
+cdk.out
+tests
+.tox
+tox.ini

--- a/l1b_lambda/Dockerfile
+++ b/l1b_lambda/Dockerfile
@@ -13,7 +13,7 @@ RUN  pip3 install mats_l1_processing-0.0.0-py2.py3-none-any.whl --target "${LAMB
 
 # Copy instrument data
 
-COPY ./calibration_data /
+COPY ./calibration_data /calibration_data
 
 # Copy function code
 COPY ./level1b/handlers/level1b.py ${LAMBDA_TASK_ROOT}

--- a/l1b_lambda/level1b/handlers/level1b.py
+++ b/l1b_lambda/level1b/handlers/level1b.py
@@ -10,7 +10,6 @@ from pandas import (  # type: ignore
     concat,
 )
 
-from mats_l1_processing import read_parquet_functions as rpf  # type: ignore
 from mats_l1_processing.instrument import Instrument  # type: ignore
 from mats_l1_processing.L1_calibrate import L1_calibrate  # type: ignore
 
@@ -100,7 +99,6 @@ def lambda_handler(event: Event, context: Context):
                     _,
                     _,
                     image_calibrated,
-                    _,
                     errors,
                 ) = L1_calibrate(ccd, instrument)
             ccd["ImageCalibrated"] = image_calibrated

--- a/l1b_lambda/level1b/handlers/level1b.py
+++ b/l1b_lambda/level1b/handlers/level1b.py
@@ -12,6 +12,7 @@ from pandas import (  # type: ignore
 
 from mats_l1_processing.instrument import Instrument  # type: ignore
 from mats_l1_processing.L1_calibrate import L1_calibrate  # type: ignore
+from mats_l1_processing import read_parquet_functions as rpf  # type: ignore
 
 
 BotoClient = Any

--- a/l1b_lambda/stacks/level1b_stack.py
+++ b/l1b_lambda/stacks/level1b_stack.py
@@ -40,7 +40,7 @@ class Level1BStack(Stack):
             code=DockerImageCode.from_image_asset("."),
             timeout=lambda_timeout,
             architecture=Architecture.X86_64,
-            memory_size=1024,
+            memory_size=3072,
             environment={
                 "L1B_BUCKET": output_bucket.bucket_name,
             },

--- a/l1b_lambda/stacks/level1b_stack.py
+++ b/l1b_lambda/stacks/level1b_stack.py
@@ -18,7 +18,6 @@ class Level1BStack(Stack):
         output_bucket_name: str,
         lambda_timeout: Duration = Duration.seconds(900),
         queue_retention_period: Duration = Duration.days(14),
-        queue_visibility_timeout: Duration = Duration.hours(12),
         **kwargs
     ) -> None:
         super().__init__(scope, id, **kwargs)
@@ -51,7 +50,7 @@ class Level1BStack(Stack):
             self,
             "Level1AQueue",
             retention_period=queue_retention_period,
-            visibility_timeout=queue_visibility_timeout,
+            visibility_timeout=lambda_timeout,
             removal_policy=RemovalPolicy.RETAIN,
             dead_letter_queue=DeadLetterQueue(
                 max_receive_count=1,

